4-projects/development.auto.example.tfvars

```diff
@@ -15,3 +15,5 @@
  */
 
 perimeter_name = "sp_d_shared_restricted_default_perimeter_????"
+
+//enable_hub_and_spoke = true
```

Comment:
Should this not go in the common tfvars file so the user does not have to set it three times?

Reply:
@rjerrems If we add it in the commons tfvars files we get a warning that we are setting a variable that does not exist when running the shared envs for bu1 and bu2 .

Reply:
aha got it, thanks!